conda_forge_tick/migrators/version.py

```diff
@@ -438,8 +432,7 @@ def _get_attempts_r(node, seen):
         @functools.lru_cache(maxsize=1024)
         def _get_attempts(node):
             if _has_solver_checks(node):
-                seen = set()
-                return _get_attempts_r(node, seen)
+                return _get_attempts_r(node, seen=set())
```

Comment:
But nobody can understand this :/

In other words: `set()` is a magic constant that is explained by passing it as a keyword argument.

Reply:
Then why change the code at all. It was already clear.

Reply:
mypy complained about a missing type annotation for `seen = set()`.
If I pass `set` as keyword argument, it can infer the type of the empty set from the function signature.

Reply:
done 9bd52fe567db0b9c3d5b0a9904d3551463edad96